src/Codeception/Module/REST.php

```diff
@@ -301,6 +301,28 @@ public function amBearerAuthenticated($accessToken)
     }
 
     /**
+     * Adds NTLM authentication via username/password.
+     * Requires Guzzle >=6.3.0 for NTLM auth option otherwise response code will always be 401.
+     *
+     * @param $username
+     * @param $password
+     * @throws ModuleException
+     * @part json
+     * @part xml
+     */
+    public function amNTLMAuthenticated($username, $password)
+    {
+        if (\GuzzleHttp\Client::VERSION < '6.2.1') {
```

Comment:
6.2.1? It would be better to use [version_compare](http://php.net/version_compare) function, just in case if 6.20.0 is ever release.